### ISSUE TITLE
"View post on Instagram" is keep loading, do no have the actual post display

### ISSUE THREAD

**Issue by Ethan-LN** [justinmahar/react-social-media-embed#33]
"View post on Instagram" is keep loading, do no have the actual post display
![Screen Shot 2023-02-06 at 14 24 39](https://user-images.githubusercontent.com/71419743/216877338-f7819889-7f7e-4012-a82d-3a3141cf41bf.png)

**Comment by shubham-spreadd** [comment]
Getting the same error for embed URLs.
Any work around?

**Comment by jmmrte** [comment]
Still an issue for you guys? Sometimes these services have hiccups and things magically work again a few hours later. Just tried on the demo site and seems to be working now.

**Comment by justinmahar** [comment]
Still an issue for you guys? Sometimes these services have hiccups and things magically work again a few hours later. Just tried on the demo site and seems to be working now.

**Comment by Ethan-LN** [comment]
> Still an issue for you guys? Sometimes these services have hiccups and things magically work again a few hours later. Just tried on the demo site and seems to be working now.

Not for me at the moment  🙈. It works on my local but not on the website I hosted on firebase.  Besides this, I have also tried using the original Instagram embed code on my HTML file.  🤕 Still， It works on my local npm, not on firebase.  I also created a public account to set up the proper Instagram post to embed into my website.  The outcomes remain the same.   Your demo site does not display the post from my browser, this is weird.

**Comment by Ethan-LN** [comment]
Hey JustinMaha. Thanks for your reply. I had to look at my browser console error. It is more likely my browser setting has blocked access to the post with Instagram. Both my chrome on MacBook and duckduckgo on mobile have the same issue. I tried the tor browser on my MacBook and safari on my mobile. Both of them display smoothly. 😆 I wonder if some browser safety settings may prevent this embed to work. I would find out a bit more details later.
